consumer/nf_discovery.go

```diff
@@ -65,7 +97,7 @@ func SendNFIntancesUDR(id string, types int) string {
 	// 	localVarOptionals.Gpsi = optional.NewString(id)
 	// }
 	fmt.Println(self.NrfUri)
-	result, err := SendNFIntances(self.NrfUri, targetNfType, requestNfType, localVarOptionals)
+	result, err := SendSearchNFInstances(self.NrfUri, targetNfType, requestNfType, &localVarOptionals)
```

Comment:
In [PCF PR 131](https://github.com/omec-project/pcf/pull/131/files#diff-6395315dbec97d6d37a98008faeb9765404d4f14165e9955a520f00bded0b78fR82), you changed `localVarOptionals` from `Nnrf_NFDiscovery.SearchNFInstancesParamOpts` to `&Nnrf_NFDiscovery.SearchNFInstancesParamOpts`, should we follow the same approach as in the [PCF PR 131](https://github.com/omec-project/pcf/pull/131) to have these similar functions/variables defined in a similar way across the different NFs?

Reply:
The changes are done to follow the same approach which are done in other NF's.